src/views/albums/Album.vue

```diff
@@ -102,6 +108,24 @@ export default {
     },
   },
   methods: {
+    ...mapActions("albums", ["read"]),
+    ...mapActions("tracks", { indexTracks: "index" }),
+    async fetchContent(newValue, oldValue) {
+      // After loading the content, the router will change the id from a string to a number
+      // but we don't actually want to load the content twice
+      if (newValue == oldValue) {
+        return;
+      }
+
+      const album = this.read(this.id);
+      const tracks = this.indexTracks(new TracksScope().album(this.id));
```

Comment:
It feels a little arbitrary what is fetched here. Why not fetch the genres of the tracks, why not fetch the labels?

Reply:
I mostly started from the scopes that are available. There isn't (yet) a scope to get genres by tracks or labels by album on the API.

Reply:
Ah, makes sense.